vault/core.go

```diff
@@ -2386,25 +2386,22 @@ func (c *Core) postUnseal(ctx context.Context, ctxCancelFunc context.CancelFunc,
 			v()
 		}
 	} else {
-		workerChans := make([]chan func(), postUnsealFuncConcurrency)
+		jobs := make(chan func())
 		var wg sync.WaitGroup
 		for i := 0; i < postUnsealFuncConcurrency; i++ {
```

Comment:
This is a neat way to use WGs to manage this that I haven't seen before.

Reply:
What a great change! This is definitely an improvement. Much easier to understand imo.